src/Tribe/Admin/Notice/Timezones.php

```diff
@@ -1,25 +1,85 @@
 <?php
+use Tribe__Date_Utils as Date;
+use Tribe__Timezones as Timezones;
+use Tribe__Events__Timezones as Event_Timezones;
+
 /**
  * Shows an admin notice for Timezones
  * (When using UTC and on TEC Pages or WordPress > General Settings)
  */
 class Tribe__Events__Admin__Notice__Timezones {
 
+	/**
+	 * Notice Slug on the user options
+	 *
+	 * @since  TBD
+	 * @var string
+	 */
+	private $slug = 'events-utc-timezone';
+
 	public function hook() {
+		$date = $this->get_current_reset_date();
+		$slug = $this->slug;
+		/**
+		 * Allows users to completely deactivate the resetting of the Day Light savings notice
+		 *
+		 * @since  TBD
+		 *
+		 * @param  bool
+		 */
+		$should_reset = apply_filters( 'tribe_events_admin_notice_daylight_savings_reset_notice', true );
```

Comment:
seems like we're not really using this boolean to truly hide the notice; shouldn't it, for example, be down in the `should_display()` method?

if that's not the goal of this, i'd recommend naming the var better and/or making the docblock comment more clear

Reply:
That is for reseting the notice or not